redstone/client.py

```diff
@@ -567,6 +571,10 @@ def create_key(self, name, payload=None, raw_payload=None, root=False):
         elif payload is not None:
             data["resources"][0]["payload"] = base64.b64encode(payload).decode("utf-8")
 
+        # creates a new key with alias name. A key can have a maximum of 5 alias names
```

Comment:
You need to check if the list is greater than 5 since there is a limit

raise a `ValueError` with a description of the error if that is the case

Reply:
Updated